src/CustomerBundle/Controller/CustomerController.php

```diff
@@ -75,29 +75,33 @@ public function addressesAction()
     /**
      * Adds an address to current customer.
      *
-     * @return Response
+     * @param Request $request
+     *
+     * @return RedirectResponse|Response
      */
-    public function addAddressAction()
+    public function addAddressAction(Request $request)
```

Comment:
This is a BC break

Reply:
@core23 Why? Request $request is inject by Controller Resolver since Symfony 2.3 
just look at:
https://symfony.com/doc/current/create_framework/http_kernel_controller_resolver.html

Reply:
or should i inject request_stock to Controller?

Reply:
> @core23 Why? Request $request is inject by Controller Resolver since Symfony 2.3
> just look at:
> https://symfony.com/doc/current/create_framework/http_kernel_controller_resolver.html

That's right, but someone could extend this Controller and the signature change will break a method override.

Reply:
> or should i inject request_stock to Controller?

You could make the parameter nullable or target the master branch.

Reply:
I change target to master. I will make another PR to 3.x branch with missing function from PR description.